signingscript/docker.d/passwords.yml

```diff
@@ -413,6 +413,12 @@ in:
              {"$eval": "AUTOGRAPH_AUTHENTICODE_SHA2_PASSWORD"},
              ["autograph_authenticode_sha2", "autograph_authenticode_sha2_stub"]
              ]
+          - ["https://autograph-external.prod.autograph.services.mozaws.net",
+             {"$eval": "AUTOGRAPH_AUTHENTICODE_EV_USERNAME"},
+             {"$eval": "AUTOGRAPH_AUTHENTICODE_EV_PASSWORD"},
+             ["autograph_authenticode_sha2"],
+             "ev"
```

Comment:
I think you need to remove the keyid here and change the format to `autograph_authenticode_ev`, plus add to `FORMAT_TO_SIGNING_FUNCTION`